README.md

```diff
@@ -158,146 +255,96 @@ Wenn der Request nicht erfolgreich verarbeitet werden konnte, liefert die Schnit
       ]
     }
 
-## Query bestesAngebot  
+## Request Format
 
-Für einen Partner wird unter Berücksichtigung seiner Handelsbeziehungen das beste Angebot ermittelt.
+Die Angaben werden als JSON im Body des Requests gesendet.  
+Die Attribute innerhalb eines Blocks können in beliebiger Reihenfolge angegeben werden.  
+Für einen erfolgreichen Request muss die Query in folgendem Format vorhanden sein (siehe auch den [Beispiel Requests](#beispiele)):
 
-### BestesAngebot: Request Format  
+**bestesAngebot** / **angebote**
 
-Die Angaben werden als JSON im Body des Requests gesendet.
-Die Attribute können in beliebiger Reihenfolge angegeben werden.  
-Die angegebenen Query-Parameter sind typisiert - ein Ausrufezeichen (!) kennzeichnet die Pflicht-Parameter.
-Die konkreten Argumente für die Anfrage werden im **variables**-Teil übergeben.  
```

Comment:
Das mit den konkreten Werten fände ich gut, wenn das auch irgendwie noch mal erklärt wird.

Reply:
hast du da eine konkrede Idee?

Reply:
mh, in deinen Beispielen ist das ja mit drin, vielleicht reicht das auch erstmal ...